kotlinpoet-metadata-specs/src/main/kotlin/com/squareup/kotlinpoet/metadata/specs/KotlinPoetMetadataSpecs.kt

```diff
@@ -920,6 +926,11 @@ private fun createThrowsSpec(
       .build()
 }
 
+private fun String.substringAfterLast(delimiters: CharArray): String {
```

Comment:
`vararg Char` is more common in the stdlib

Reply:
970994b